site_source_files/content/blog/2021/moving-to-libera-chat.md

```diff
@@ -0,0 +1,26 @@
++++
+Description = "Moving to Libera.Chat"
+Title = "IRC channels are moving to Libera.chat"
+Date = 2021-05-20
+Author = "David Burns"
+AuthorLink = "https://twitter.com/AutomatedTester"
+tags = ["chat", "selenium", "freenode", "libera.chat", "irc"]
+categories = ["general"]
++++
+
+Yesterday, 19 May 2021, all the staff of Freenode, the IRC network, [collectively
+resigned as staff because of what they felt was a hostile takeover](https://fuchsnet.ch/freenode-resign-letter.txt).
+
+Freenode has always been seen as the home of Free and Open Software, and one of the
+reasons why the Selenium Project felt so at home. To maintain this openness, and inclusivity,
```

Comment:
Nit: "home" twice in the same sentence is clumsy. Also, it's a little odd for the project to feel at home on an IRC network.

"Freenode has always been seen as the home of Free and Open Software, so it was natural that you could find us on the #selenium channel there"?